sdcm/utils/raft/common.py

```diff
@@ -78,11 +79,12 @@ class NodeBootstrapAbortManager:
     INSTANCE_START_TIMEOUT = 600
     SUCCESS_BOOTSTRAP_TIMEOUT = 3600
 
-    def __init__(self, bootstrap_node: BaseNode, verification_node: BaseNode):
+    def __init__(self, bootstrap_node: BaseNode, verification_node: BaseNode, actions_log: ActionLogger):
         self.bootstrap_node = bootstrap_node
         self.verification_node = verification_node
         self.db_cluster: BaseScyllaCluster = verification_node.parent_cluster
         self.monitors: BaseMonitorSet = self.verification_node.test_config.tester_obj().monitors
+        self.actions_log = actions_log
```

Comment:
I am not sure we need action logging in this class. This will cause a lot of messages when setting up new DC or growing the cluster